cmd/api/handlers-environments.go

```diff
@@ -0,0 +1,55 @@
+package main
+
+import (
+	"log"
+	"net/http"
+
+	"github.com/gorilla/mux"
+	"github.com/jmpsec/osctrl/pkg/settings"
+	"github.com/jmpsec/osctrl/pkg/utils"
+)
+
+// GET Handler for single JSON environment
```

Comment:
Do you mean this API is used to validate if an environment exists in the backend?

Reply:
Correct, and returns all data for that environment. Does it make more sense to have a verb to check on a specific environment, passing the name, and return just a boolean?